cmake/templates/oneDPLConfig.cmake.in

```diff
@@ -75,12 +75,24 @@ if (EXISTS "${_onedpl_headers}")
         endif()
 
         if (NOT ONEDPL_PAR_BACKEND OR ONEDPL_PAR_BACKEND STREQUAL "openmp")  # Handle OpenMP backend
+            set(_openmp_minimum_version 4.5)
             find_package(OpenMP)
-            if (NOT OpenMP_CXX_FOUND AND ONEDPL_PAR_BACKEND STREQUAL "openmp")  # If OpenMP backend is requested explicitly, but not supported.
+            if (OpenMP_CXX_FOUND)
+                if (OpenMP_CXX_VERSION VERSION_LESS _openmp_minimum_version)
+                    message(STATUS "oneDPL: OpenMP version (${OpenMP_CXX_VERSION}) is less than minimum supported version (${_openmp_minimum_version}), disable openMP" )
+                    set (_openmp_found_valid FALSE)
+                else()
+                    set (_openmp_found_valid TRUE)
+                endif()
+            else()
+                set (_openmp_found_valid FALSE)
+            endif()
```

Comment:
You can also slightly refactor the code to reduce its amount:

- Replace `VERSION_LESS` with `VERSION_GREATER_EQUAL` and remove all `else` branches. 
- Run only `set (_openmp_found_valid TRUE)`; If `_openmp_found_valid` is not defined, it should be treated as false.
- Enhance `message(STATUS "oneDPL: ONEDPL_PAR_BACKEND=${ONEDPL_PAR_BACKEND} requested, but not supported")` with the version information.

Reply:
`message(STATUS "oneDPL: ONEDPL_PAR_BACKEND=${ONEDPL_PAR_BACKEND} requested, but not supported")`  is also used when OpenMP is not found for any version.  I formed it like this to reuse that error return branch rather than duplicating it.  Version information isn't guaranteed to exist at that line.

The other suggestions work for me though.

Reply:
I think removing the `set(_openmp_found_valid FALSE)` makes sense and I've delete them as well as an `else` branch which was only serving one. However, I believe we only want a version comparison message when OpenMP was found and is less than the version required. This negates the advantage from switching to `VERSION_GREATER_EQUAL`.

Reply:
Thanks, it looks simpler now